release-notes/Web_apps/Web_apps_Feature_Release_10.4/Web_apps_Feature_Release_10.4.6.md

```diff
@@ -140,14 +140,14 @@ From now on, a *Spectrum analyzer* component will indicate in a clearer way that
 
 When an *Alarm table* component is configured to show alarms in a sliding window, from now on, you will be able to specify the following limits:
```

Comment:
As far as I understand it, this RN adds new limits to what the user can configure, but this sentence seems to suggest that the user can specify those limits. Could you check and adjust if necessary?